core/src/main/java/bio/terra/pearl/core/dao/search/EnrolleeSearchExpressionDao.java

```diff
@@ -92,6 +93,16 @@ public EnrolleeSearchExpressionResult map(ResultSet rs, StatementContext ctx) th
                     );
                 }
 
+                if (columnName.startsWith("task_") && columnName.endsWith("_created_at")) {
```

Comment:
this code feels like it would be cleaner with a regex -- you're essentially looking for a match to `/^task_(.+)_created_at$/`